lib/fluent/plugin/in_monitor_agent.rb

```diff
@@ -330,7 +282,7 @@ def all_plugins
     # from the plugin `pe` recursively
     def self.collect_children(pe, array=[])
       array << pe
-      if pe.is_a?(MultiOutput) && pe.respond_to?(:outputs)
```

Comment:
You should do `pe.is_a?(Fluent::Plugin::MultiOutput) || pe.is_a?(Fluent::MultiOutput)` (both respond to `#otuputs`).